README.md

```diff
@@ -50,7 +50,7 @@ python -m pip install -e .[testing] -U
 
 ```sh
 python -m pip install flit
-flit install
+flit install -s
```

Comment:
nice one! (reference for others: `-s` is for symlinking the package so it doesn't need re-installing. Very useful for local dev indeed)

Reply:
Does anyone develop with just `flit install` at all?

Reply:
I may know a person... _whistles nonchalantly_